public-ui/src/app/styles/buttons.css

```diff
@@ -1,84 +1,107 @@
-button, a.btn {
-  font-size: 12px;
-  font-weight: 500;
-  font-family: "Gotham HTF",open-sans,sans-serif;;
-  line-height:14px;
+.btn-container{
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.db-btn {
```

Comment:
Indentation space in btn-container and db-btn style classes seems to be different.